CMakeLists.txt

```diff
@@ -421,6 +422,14 @@ if (ENABLE_ENCRYPTION)
 	add_definitions(-DSRT_ENABLE_ENCRYPTION)
 	message(STATUS "ENCRYPTION: ENABLED, using: ${SSL_REQUIRED_MODULES}")
 	message (STATUS "SSL libraries: ${SSL_LIBRARIES}")
+
+	if (ENABLE_AEAD_PREVIEW)
```

Comment:
Rather throw an error if encryption isn't enabled.

Encryption is enabled by default, so a user would have to explicitly disable it. It's unlikely that someone would consciously enable AEAD and expect that encryption is off.

Reply:
Renamed to `ENABLE_AEAD_API_PREVIEW`.
The build option only enables the API. To have the feature available, the OpenSSL EVP needs to be enabled.
Not sure if checking all those combinations makes sense, given the option will be removed in v1.6.0.
OTOH it might be user-friendly to do so.

Reply:
Done.